core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/FulltextSearchQueryUtils.scala

```diff
@@ -0,0 +1,151 @@
+package edu.uci.ics.texera.web.resource.dashboard
+
+import edu.uci.ics.texera.web.model.jooq.generated.Tables._
+import edu.uci.ics.texera.web.resource.dashboard.DashboardResource.SearchQueryParams
+import org.jooq.{Condition, Field, OrderField}
+import org.jooq.impl.DSL.{condition, noCondition}
+
+import java.sql.Timestamp
+import java.text.{ParseException, SimpleDateFormat}
+import java.util.concurrent.TimeUnit
+import scala.jdk.CollectionConverters.CollectionHasAsScala
+
+object FulltextSearchQueryUtils {
```

Comment:
I thought we were going to have three sub-classes for implementing full-text from different databases (MySQL, Postgres, and SQLite). I do not see how to extend a function to write SQLite full-text search in the current framework.

Reply:
it's not necessary for now. We can do it later.